gen3utils/manifest/manifest_validator.py

```diff
@@ -132,7 +132,7 @@ def get_manifest_version(manifest_versions, service):
             # calling get_manifest_version()! it should never happen
             service_line = manifest_versions[service]
             service_version = service_line.split(":")[1]
-            if version_is_branch(service_version):
+            if version_is_branch(service_version, release_tag_are_branches):
```

Comment:
can you explain why we need this change

Reply:
Without this change, when we try to parse `tube: 2020.10` here https://github.com/uc-cdis/gen3utils/pull/36/files/bf2d0e01a3c35695935186e8e6276b7fa9fef9dd#diff-dd2f4895d82dd057d40cc78d293dee1040b66ec665d257326fab07578953f192R327 we get back a string, not a version. So I added the parameter to make `get_manifest_version` optionally treat the monthly release branches as versions.

Reply:
ok i see, but then you'll need more complex logic in the `if` 😕 

`if tube_version >= version.parse("0.4.0") or tube_version >= version.parse("2020.10")` would be True for version 2020.03

Reply:
this is why monthly releases are so hard to work this in this repo

Reply:
oh, woof. Good catch. hmmm... I could add a quick `is_monthly_release_tag` fn that just checks to see if a version is < `2019.0.0`. What do you think?

Reply:
there's probably a way to check if the major number of the `version` is >= 2020 🤔